packages/node_modules/@ciscospark/spark-core/test/unit/spec/lib/batcher.js

```diff
@@ -158,7 +158,7 @@ describe(`spark-core`, () => {
 
           const promises = [];
           // eslint-disable-next-line no-unmodified-loop-condition
-          for (let i = 0; i < BATCHER_MAX_CALLS + 1; i++) {
```

Comment:
I'm not sure what you mean. I changed it from `i < BATCHER_MAX_CALLS + 1` to `i < BATCHER_MAX_CALLS`. Did you want me to make it `i <= BATCHER_MAX_CALLS + 1`?